### PR TITLE
[Critical Bug]Data inconsistency will occur when java.lang.Error is throwen. 

### PR DESCRIPTION
**Pull request by daisuke-yoshimoto**
https://github.com/Activiti/Activiti/issues/1525

### CODE REVIEW

**Comment by Salaboy** [comment]
@erdemedeiros I'm approving this one, but please review and merge on monday. It looks like a good fix.

**Comment by Salaboy** [comment]
@daisuke-yoshimoto in the mean time, you can fix some indentation problems reported by codacy: https://www.codacy.com/app/Activiti/Activiti/pullRequest?prid=1026187

**Comment by daisuke-yoshimoto** [comment]
@Salaboy Thank you for your review.
I fixed indentation problems by applying Activiti 7 code style.

File: activiti-engine/src/main/java/org/activiti/engine/impl/interceptor/CommandContextInterceptor.java [hunk 1]
```diff
@@ -62,7 +62,7 @@ public CommandContextInterceptor(CommandContextFactory commandContextFactory, Pr
 
       return next.execute(config, command);
 
-    } catch (Exception e) {
+    } catch (Throwable e) {
```
Comment (erdemedeiros) [inline activiti-engine/src/main/java/org/activiti/engine/impl/interceptor/CommandContextInterceptor.java]: :+1:

**Pull request merged**